docs/spark-queries.md

```diff
@@ -318,12 +318,15 @@ To show a table's current partitions:
 SELECT * FROM prod.db.table.partitions
 ```
 
-| partition | record_count | file_count |
-| -- | -- | -- |
-|  {20211001, 11}|           1|         1|
-|  {20211002, 11}|           1|         1|
-|  {20211001, 10}|           1|         1|
-|  {20211002, 10}|           1|         1|
+| partition | record_count | file_count | spec_id |
+| -- | -- | -- | -- |
+|  {20211001, 11}|           1|         1|         0|
+|  {20211002, 11}|           1|         1|         0|
+|  {20211001, 10}|           1|         1|         0|
+|  {20211002, 10}|           1|         1|         0|
+
+Note:
+If this table is non-partitioned, it will contain only the record_count and file_count columns.
```

Comment:
Actually sorry about it, I re-read it and can you make one more fix?    I just realized "this table... it" will be wrong because we mean different tables (data table and metadata table)

"If the table is unpartitioned, the partitions table will contain only the record_count and file_count columns."

Reply:
Thanks, addressed it.